start_cerberus.py

```diff
@@ -144,11 +156,26 @@ def main(cfg):
                     logging.info("%s: %s", namespace, failures)
                 if slack_integration:
                     failed_namespaces = ", ".join(list(failed_pods_components.keys()))
-                    slackcli.post_message_in_slack("%sIn iteration %d, Cerberus found issues in "
-                                                   "namespaces: %s. Hence, setting the go/no-go "
-                                                   "signal to false. Have a look at the report "
-                                                   "below!" % (cluster_info, iteration,
-                                                               failed_namespaces))
+                    valid_cops = slackcli.get_channel_members()['members']
+                    if cop_slack_member_ID in valid_cops:
+                        # If the cop assigned is a member of the slack channel, tag the cop
+                        # while reporting every failure
+                        slackcli.post_message_in_slack("<@" + cop_slack_member_ID + "> %sIn "
+                                                       "iteration %d, cerberus found issues in "
+                                                       "namespaces: %s. Hence, setting the go/"
+                                                       "no-go signal to false. Have a look "
+                                                       "at the report below!" % (cluster_info,
+                                                                                 iteration,
+                                                                                 failed_namespaces))
+                    else:
+                        # If a cop isn't assigned for the day, tag everyone by using @here
+                        # while reporting every failure
+                        slackcli.post_message_in_slack("@here %sIn iteration %d, cerberus found "
+                                                       "issues in namespaces: %s. Hence, setting "
+                                                       "the go/no-go signal to false. Have a look "
+                                                       "at the report below!" % (cluster_info,
+                                                                                 iteration,
+                                                                                 failed_namespaces))
                     slackcli.post_file_in_slack()
```

Comment:
Better to avoid posting the report on slack to avoid overloading slack with big files when cerberus is running for long durations/days like @rsevilla87 and @jtaleric suggested. For now, we can just print the failed iteration and mention the location of the full report. @yashashreesuresh @rsevilla87 @jtaleric sounds good?

Reply:
Yeah, sounds good :) . Let's merge this PR. In case we need more info in the future, we may add a flag to support sending previous iterations, take it as a suggestion that could be applied if eventually required.
Nice job @yashashreesuresh

Reply:
Thank you! @rsevilla87